src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -325,8 +325,8 @@ const buildWhereClause = ({ schema, query, index }) => {
       const point = fieldValue.$nearSphere;
       const distance = fieldValue.$maxDistance;
       const distanceInKM = distance * 6371 * 1000;
-      patterns.push(`ST_distance_sphere($${index}:name::geometry, POINT($${index + 1}, $${index + 2})::geometry) <= $${index + 3}`);
-      sorts.push(`ST_distance_sphere($${index}:name::geometry, POINT($${index + 1}, $${index + 2})::geometry) ASC`)
+      patterns.push(`ST_distance_sphere($${index}:name::geometry, POINT($${index + 2}, $${index + 1})::geometry) <= $${index + 3}`);
+      sorts.push(`ST_distance_sphere($${index}:name::geometry, POINT($${index + 2}, $${index + 1})::geometry) ASC`)
       values.push(fieldName, point.longitude, point.latitude, distanceInKM);
       index += 4;
```

Comment:
I may have incomplete understanding in what that `ST_distance_sphere` does, but just to make sure, the `:name` formatting results in the value being wrapped into double quotes, as in `ST_distance_sphere("value", `.

Are you guys sure this is what the function actually expects? 'Cos I'm not sure...